docs/development/getting-started-remotely.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Deploying Registry Cache Extension Remotely
+description: Learn how to set up a development environment using own Seed clusters on an existing Kubernetes cluster
+---
+
+# Deploying Registry Cache Extension Remotely
```

Comment:
I am not sure if `remotely` is the right term? This guide is about deploying the extension is gardener's local setup with provider extensions. 

Something like:
```suggestion
# Deploying Registry Cache Extension in Gardener's Local Setup with Provider Extensions
```

would be more meaningful to me.